src/styles/index.css

```diff
@@ -105,6 +105,8 @@
     --p-color-input-checked-border: var(--p-color-selected);
     --p-color-input-checked-text: var(--p-color-text-default);
 
+    --p-color-input-checkbox-indeterminate-bg-img: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='' viewBox='0 0 16 16'%3e%3cpath stroke='white' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M4 8h8'/%3e%3c/svg%3e");
```

Comment:
is this really weird that this is prefixed with `--p-color`... 

this feels like an appropriate place to put this but every single variable in this area is prefixed with `--p-color` 😶‍🌫️

Reply:
I don't think it needs to color prefix. 🤷